apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -76,12 +98,12 @@ var GameLabVisualizationColumn = React.createClass({
             height={GAME_HEIGHT}
             onMouseMove={this.onMouseMove}
           >
+            <GridOverlay show={this.props.showGrid} />
```

Comment:
@Bjvanminnen I would expect that passing in this prop would update this component but since it's in a protected div, is there something else I need to do to make this work?
I can log and see that redux state has the right value, but my component does not.